app/components/DR/ButtonUpDrawer.js

```diff
@@ -180,7 +183,12 @@ export default class BottomUpPanel extends Component {
   };
 
   toggle = () => {
```

Comment:
Remove `languages.` from the t
```suggestion
  toggle = () => {
  const {t} = this.props;
```